prboom2/src/wi_stuff.c

```diff
@@ -567,6 +569,13 @@ void WI_drawEL(void)
 		// CPhipps - patch drawing updated
 		V_DrawNamePatch((320 - V_NamePatchWidth(lname)) / 2, y, FB,
 			lname, CR_DEFAULT, VPT_STRETCH);
+
+		y += (5 * V_NamePatchHeight(lname)) / 4;
+	}
+
+	if (wbs->nextmapinfo && wbs->nextmapinfo->author)
```

Comment:
The spec stipulates that the author name is only printed when the levelpic doesn't exist and we are printing the level name (the above branch). In a lot of wads, the organizer includes the author in the patch along with the title.

Reply:
@rfomin could you apply the change requested by @kraflab above? This would be the last remaining change I would like to merge into the prboom-plus code base before nailing is coffin.

Reply:
Done in https://github.com/coelckers/prboom-plus/pull/436/commits/1bb0a74a57d27571efefb6bf12620e55ac1c045d